spec/controllers/admin/users_controller_spec.rb

```diff
@@ -18,6 +18,19 @@
       expect(response).to render_template :index
     end
   end
+  describe 'PATCH #togle_confirmation' do
+    it 'confirms user' do
+      unconfirmed_user = create(:user, email: 'unconfirmed_user@osem.io', confirmed_at: nil)
+      patch :toggle_confirmation, id: unconfirmed_user.id, user: { to_confirm: 'true' }
+      confirmed_user = User.find(unconfirmed_user.id)
```

Comment:
You can delete that line, and possibly name the variable `user_to_confirm` or something like that, that makes logical sense